sdk/cosmosdb/cosmos/package.json

```diff
@@ -11,28 +11,33 @@
     "database",
     "cloud"
   ],
-  "version": "2.1.5",
+  "version": "3.0.0-15",
   "author": "Microsoft Corporation",
-  "main": "./lib/src/index.js",
-  "types": "./lib/src/index.d.ts",
+  "main": "./dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./dist/index.d.ts",
   "engine": {
     "node": ">=6.0.0"
   },
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build:dev": "npm run check-format && npm run lint && npm run build:test",
-    "build:test": "npm run compile:dev && npm run webpack:dev",
-    "build": "npm run compile && npm run webpack",
-    "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\"",
-    "clean": "rimraf lib",
-    "compile:dev": "echo Using TypeScript && tsc --version && tsc --pretty",
-    "compile": "echo Using TypeScript && tsc --version && tsc -p tsconfig.prod.json --pretty",
-    "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\"",
-    "integration-test:browser": "echo skipped",
-    "integration-test:node": "mocha -r ./src/test/common/setup.ts ./lib/src/test/ --recursive --timeout 100000 --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- -i -g .*ignore.js",
+    "build:test": "npm run compile",
+    "build": "npm run compile && node writeSDKVersion.js && npm run bundle && npm run bundle-types",
+    "bundle": "rollup -c",
+    "bundle-types": "npm run extract-api",
+    "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\"",
+    "ci": "npm run build && npm run test -- --forbid-only && node consumer-test.js",
```

Comment:
Fine to keep this script but the CI will never run it so maybe it should have a new name.